bindings/python/src/OpenSpaceToolkitAstrodynamicsPy/Trajectory/Model.cpp

```diff
@@ -68,7 +68,7 @@ inline void OpenSpaceToolkitAstrodynamicsPy_Trajectory_Model(pybind11::module &a
             &BaseModel::calculateStatesAt,
             arg("instants"),
             R"doc(
-                Calculate the states at given instants.
+                Calculate the states at given instants. This method is preferred over calling `calculate_state_at` multiple times since there is a performance gain.
```

Comment:
```suggestion
                Calculate the states at given instants. It can be more performant than looping `calculate_state_at` for multiple instants.
```